task-2/src/components/Formik/movieValidationSchema.js

```diff
@@ -4,7 +4,7 @@ export const movieValidationSchema = Yup.object().shape({
 	title: Yup.string().required('Title is required'),
 	release_date: Yup.string().required('Release Date is required'),
 	poster_path: Yup.string().required('Image Url is required'),
-	genres: Yup.string().required('Genre is required'),
+	// genres: Yup.string().required('Genre is required'),
```

Comment:
Consider removing commented pieces of code.

Reply:
Resolved.